### PR TITLE
Hardware .json docs

### PR DESCRIPTION
**Pull request by gerth2**
closes https://github.com/PhotonVision/photonvision-docs/issues/120

### CODE REVIEW

**Comment by gerth2** [comment]
Bonus - ran my spell-checker on other files, and lumped it into one PR, because I'm that organized.

Can split if we care.

File: source/conf.py [hunk 1]
```diff
@@ -115,4 +115,4 @@
 
 suppress_warnings = ['epub.unknown_project_files']
 
-sphinx_tabs_valid_builders = ['epub', 'linkcheck']
+sphinx_tabs_valid_builders = ['epub', 'linkcheck', 'lint']
```
Comment (mdurrani808) [inline source/conf.py:118]: Assuming that this didn't work, I would delete this for now while we look for a working solution.

File: source/docs/hardware/customhardware.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
+Deploying on Custom Hardware
+============================
+
+Configuration
+-------------
+
+By default, PhotonVision attempts to make minimal assumptions of the hardware it runs on. However, it may be configured to enable more functionality.
```
Comment (mdurrani808) [inline source/docs/hardware/customhardware.rst:7]: "to enable more functionality including LED control, performance metrics, and brand customization"
Thoughts?

File: source/docs/hardware/customhardware.rst [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+``hardwareConfig.json`` is the location for this configuration. It is included when settings are exported, and can be uploaded as part of a .zip, or on its own.
+
+LED Support
+-----------
+
+For Raspberry-Pi based hardware, PhotonVision can use `PiGPIO <https://abyz.me.uk/rpi/pigpio/>`_ to control IO pins. The mapping of which pins control which LED's is part of the hardware config.
+
+.. tabs::
```
Comment (mdurrani808) [inline source/docs/hardware/customhardware.rst:16]: Could you change these tabs to sphinx-design tabs? We will be moving over to these once the relevant PR gets merged.
https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html

File: source/docs/hardware/customhardware.rst [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+.. note:: No hardware boards with status RGB LED pins or non-dimming LED's have been tested yet. Please reach out to the development team if these features are desired, they can assist with configuration and testing.
+
+Hardware Interaction Commands
+-----------------------------
+
+For Non-Raspberry-Pi hardware, users must provide valid hardware-specific commands for some parts of the UI interaction (including performance metrics, and executing system restarts).
+
+Leaving a command blank will disable the associated functionality.
+
+.. tabs::
```
Comment (mdurrani808) [inline source/docs/hardware/customhardware.rst:40]: See tabs thing

File: source/docs/hardware/customhardware.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+      }
+
+.. note:: These settings have no effect if PhotonVision detects it is running on a Raspberry Pi. See `the MetricsBase class <https://github.com/PhotonVision/photonvision/blob/dbd631da61b7c86b70fa6574c2565ad57d80a91a/photon-core/src/main/java/org/photonvision/common/hardware/metrics/MetricsBase.java>`_ for the commands utilized.
+
+Known Camera FOV
+----------------
+
+If your hardware contains a camera with a known field of vision, it can be entered into the hardware configuration. This will prevent users from editing it in the GUI.
+
+.. tabs::
```
Comment (mdurrani808) [inline source/docs/hardware/customhardware.rst:60]: tabs

File: source/docs/hardware/customhardware.rst [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+      {
+        "vendorFOV" : 98.9
+      }
+
+Cosmetic & Branding
+-------------------
+
+To help differentiate your hardware from other solutions, some customization is allowed.
+
+.. tabs::
```
Comment (mdurrani808) [inline source/docs/hardware/customhardware.rst:72]: Tabs

**Review by mdurrani808 — CHANGES_REQUESTED** [review]

**Comment by mdurrani808** [comment]
Also: thanks for the spell checking! All fine in this PR, shouldn't be anything structurally important.

File: source/conf.py (hunk 1 above)
Comment (gerth2) [reply source/conf.py:118]: yeeted.

File: source/docs/hardware/customhardware.rst (hunk 3 above)
Comment (gerth2) [reply source/docs/hardware/customhardware.rst:16]: done, and looks pretty

File: source/docs/hardware/customhardware.rst (hunk 2 above)
Comment (gerth2) [reply source/docs/hardware/customhardware.rst:7]: got something close to that which I like

**Review by mcm001 — APPROVED** [review]

**Review by mdurrani808 — APPROVED** [review]

**Pull request merged**